<style type='text/css'>/* <![CDATA[ */ table.table {border-width: 0 0 1px 1px; border-spacing: 0;border-collapse: collapse;} th,td{border-color: #600;border-style: solid; margin: 0;padding: 4px;border-width: 1px 1px 1px 1px;} table.noborder, th.noborder, td.noborder{border: 0px solid black;}/* ]]> */</style><script language='javascript' type='text/javascript'><!--
function popitup(url) {	newwindow=window.open(url,'name','height=75,width=400');	if (window.focus) {newwindow.focus()}	return false;}// --></script>
<center><table class="noborder"><th bgcolor='#A8CFFF' colspan=3 class="noborder"><font size='5'>GEM/KMAC results: output10both_1</font></th>
<tr><td valign='top' class="noborder"><b>Binding Event calling</b>:<p><a href='output10both_1.GEM_events.txt'>Significant Events</a>&nbsp;&nbsp;: 696<br><a href='output10both_1.GEM_insignificant.txt'>Insignificant Events</a>: 270<br><a href='output10both_1.GEM_filtered.txt'>Filtered Events</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: 39</td>
<td valign='top' class="noborder">Read distribution<br><img src='output10both.all.Read_Distributions.png' width='350'><hr></td>
<td valign='top' class="noborder"><b>Motif Discovery</b>:<p><p>Total number of sequences: 600+/600-<br><ul><li>KSM files: <a href='output10both_1.m0.KSM.txt'>m0</a><li><a href='output10both_1.KSM_Alignements.txt'>KSM alignment file</a><li><a href='output10both_1.all.PFM.txt'>All motif PFMs</a></ul></td></tr><table></center>
<table border=1 class="table"><th>Motif ID</th><th>High scoring k-mers of the KSM motif</th><th>KSM motif</th><th>Aligned bound sequences</th><th>PWM and spatial distribution</th>
<tr><td align='center'>m0</td><td><table border=1 class="table"><th>K-mer</th><th>Offset</th><th>Pos Hit</th><th>Neg Hit</th><th>HGP</th><tr><td><b><font size='4' face='Courier New'>--<font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font></font></b></td><td>-3</td><td>107</td><td>9</td><td>-25.7</td></tr><tr><td><b><font size='4' face='Courier New'>---<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='orange'>G</font></font></b></td><td>-2</td><td>76</td><td>9</td><td>-16.1</td></tr><tr><td><b><font size='4' face='Courier New'><font color='red'>T</font><font color='grey'>N</font><font color='grey'>N</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font></font></b></td><td>-5</td><td>63</td><td>3</td><td>-17.6</td></tr><tr><td><b><font size='4' face='Courier New'>--<font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='red'>T</font><font color='orange'>G</font></font></b></td><td>-3</td><td>58</td><td>20</td><td>-5.8</td></tr><tr><td><b><font size='4' face='Courier New'>---<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='green'>A</font></font></b></td><td>-2</td><td>56</td><td>6</td><td>-12.6</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='orange'>G</font><font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font></font></b></td><td>-4</td><td>56</td><td>7</td><td>-11.9</td></tr><tr><td><b><font size='4' face='Courier New'>---<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='blue'>C</font></font></b></td><td>-2</td><td>56</td><td>8</td><td>-11.2</td></tr><tr><td><b><font size='4' face='Courier New'>---<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='red'>T</font><font color='orange'>G</font><font color='red'>T</font></font></b></td><td>-2</td><td>55</td><td>14</td><td>-7.4</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='green'>A</font><font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font></font></b></td><td>-4</td><td>47</td><td>15</td><td>-5.3</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font></font></b></td><td>-4</td><td>47</td><td>13</td><td>-6.3</td></tr></table></td><td valign='top'><a href='output10both_1.m0.KSM.txt'><img src='output10both_1.m0.KSM.png' height='300' width='200'></a><br>3.29, hit=115+/3-, auc=40.5</td><td valign='top' align='center'><img src='output10both_1.m0.sequenceHits.png' height='300' width='210'></td><td align='center' valign='bottom'><img src='output10both_1.m0.PWM.png'><br><img src='output10both_1.m0.PWM_rc.png'><br><br><a href='output10both_1.Spatial_dist.m0_m0.txt'><img src='output10both_1.Spatial_dist.m0_m0.png' width='220'></a><p>8.72/12.49, hit=337+/96-, auc=30.1<br></td></tr></table></td></tr></table><p>
Options:
--d /home/bgp01/methylation/programs/gem/Read_Distribution_default.txt --g /home/bgp01/methylation/data/commonData/genome.index.txt --genome /home/bgp01/methylation/data/commonData/Athchrs/ --s 150000000 --expt MYCH7direct1MockClean10.bam --ctrl /home/bgp01/methylation/data/tfs/Input/amplified/1/Mock/Inputamplified1MockClean10.bam --out /home/bgp01/methylation/data/pruebas/output10both/ --f SAM --outNP --range 200 --smooth 0 --mrc 1 --fold 2 --q 1.301029996 --k_min 6 --k_max 20 --k_seqs 600 --k_neg_dinu_shuffle --pp_nmotifs 1 --t 12 <p>