<style type='text/css'>/* <![CDATA[ */ table.table {border-width: 0 0 1px 1px; border-spacing: 0;border-collapse: collapse;} th,td{border-color: #600;border-style: solid; margin: 0;padding: 4px;border-width: 1px 1px 1px 1px;} table.noborder, th.noborder, td.noborder{border: 0px solid black;}/* ]]> */</style><script language='javascript' type='text/javascript'><!--
function popitup(url) {	newwindow=window.open(url,'name','height=75,width=400');	if (window.focus) {newwindow.focus()}	return false;}// --></script>
<center><table class="noborder"><th bgcolor='#A8CFFF' colspan=3 class="noborder"><font size='5'>GEM/KMAC results: GEMout_1</font></th>
<tr><td valign='top' class="noborder"><b>Binding Event calling</b>:<p><a href='GEMout_1.GEM_events.txt'>Significant Events</a>&nbsp;&nbsp;: 2<br><a href='GEMout_1.GEM_insignificant.txt'>Insignificant Events</a>: 1<br>Filtered Events: 0</td>
<td valign='top' class="noborder">Read distribution<br><img src='GEMout.all.Read_Distributions.png' width='350'><hr></td>
<td valign='top' class="noborder"><b>Motif Discovery</b>:<p><p>Total number of sequences: 3+/3-<br><ul><li>KSM files: <a href='GEMout_1.m0.KSM.txt'>m0</a><li><a href='GEMout_1.KSM_Alignements.txt'>KSM alignment file</a><li><a href='GEMout_1.all.PFM.txt'>All motif PFMs</a></ul></td></tr><table></center>
<table border=1 class="table"><th>Motif ID</th><th>High scoring k-mers of the KSM motif</th><th>KSM motif</th><th>Aligned bound sequences</th><th>PWM and spatial distribution</th>
<tr><td align='center'>m0</td><td><table border=1 class="table"><th>K-mer</th><th>Offset</th><th>Pos Hit</th><th>Neg Hit</th><th>HGP</th><tr><td><b><font size='4' face='Courier New'><font color='green'>A</font><font color='green'>A</font><font color='green'>A</font><font color='green'>A</font><font color='red'>T</font><font color='red'>T</font><font color='red'>T</font><font color='red'>T</font></font></b></td><td>-18</td><td>1</td><td>0</td><td>-Infinity</td></tr></table></td><td valign='top'><a href='GEMout_1.m0.KSM.txt'><img src='GEMout_1.m0.KSM.png' height='300' width='200'></a><br>1.19, hit=1+/0-, auc=36.7</td><td valign='top' align='center'><img src='GEMout_1.m0.sequenceHits.png' height='300' width='240'></td><td align='center' valign='bottom'><img src='GEMout_1.m0.PWM.png'><br><img src='GEMout_1.m0.PWM_rc.png'><br><br><a href='GEMout_1.Spatial_dist.m0_m0.txt'><img src='GEMout_1.Spatial_dist.m0_m0.png' width='220'></a><p>7.19/7.19, hit=1+/0-, auc=33.3<br></td></tr></table></td></tr></table><p>
Options:
--d /home/bgp01/methylation/programs/gem/Read_Distribution_default.txt --g /home/bgp01/methylation/data/commonData/genome.index.txt --genome /home/bgp01/methylation/data/commonData/Athchrs/ --s 150000000 --expt ERF1direct1Mock.bam --ctrl /home/bgp01/methylation/data/tfs/Input/amplified/1/Mock/Inputamplified1Mock.bam --out /home/bgp01/methylation/data/tfs/ERF1/direct/1/Mock/GEMout --f SAM --outNP --range 200 --smooth 0 --mrc 1 --fold 2 --q 1.301029996 --k_min 6 --k_max 20 --k_seqs 600 --k_neg_dinu_shuffle --pp_nmotifs 1 --t 10 <p>