<style type='text/css'>/* <![CDATA[ */ table.table {border-width: 0 0 1px 1px; border-spacing: 0;border-collapse: collapse;} th,td{border-color: #600;border-style: solid; margin: 0;padding: 4px;border-width: 1px 1px 1px 1px;} table.noborder, th.noborder, td.noborder{border: 0px solid black;}/* ]]> */</style><script language='javascript' type='text/javascript'><!--
function popitup(url) {	newwindow=window.open(url,'name','height=75,width=400');	if (window.focus) {newwindow.focus()}	return false;}// --></script>
<center><table class="noborder"><th bgcolor='#A8CFFF' colspan=3 class="noborder"><font size='5'>GEM/KMAC results: conocho_2</font></th>
<tr><td valign='top' class="noborder"><b>Binding Event calling</b>:<p><a href='conocho_2.GEM_events.txt'>Significant Events</a>&nbsp;&nbsp;: 3459<br><a href='conocho_2.GEM_insignificant.txt'>Insignificant Events</a>: 3233<br><a href='conocho_2.GEM_filtered.txt'>Filtered Events</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;: 19</td>
<td valign='top' class="noborder">Read distribution<br><img src='conocho.all.Read_Distributions.png' width='350'><hr></td>
<td valign='top' class="noborder"><b>Motif Discovery</b>:<p><p>Total number of sequences: 600+/600-<br><ul><li>KSM files: <a href='conocho_2.m0.KSM.txt'>m0</a> <a href='conocho_2.m1.KSM.txt'>m1</a><li><a href='conocho_2.KSM_Alignements.txt'>KSM alignment file</a><li><a href='conocho_2.all.PFM.txt'>All motif PFMs</a></ul></td></tr><table></center>
<table border=1 class="table"><th>Motif ID</th><th>High scoring k-mers of the KSM motif</th><th>KSM motif</th><th>Aligned bound sequences</th><th>PWM and spatial distribution</th>
<tr><td align='center'>m0</td><td><table border=1 class="table"><th>K-mer</th><th>Offset</th><th>Pos Hit</th><th>Neg Hit</th><th>HGP</th><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='red'>T</font></font></b></td><td>-4</td><td>147</td><td>9</td><td>-39.5</td></tr><tr><td><b><font size='4' face='Courier New'><font color='blue'>C</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font></font></b></td><td>-5</td><td>109</td><td>8</td><td>-27.6</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='grey'>N</font><font color='grey'>N</font><font color='green'>A</font></font></b></td><td>-4</td><td>102</td><td>0</td><td>-34.5</td></tr><tr><td><b><font size='4' face='Courier New'>--<font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='red'>T</font><font color='blue'>C</font></font></b></td><td>-3</td><td>95</td><td>14</td><td>-18.0</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='green'>A</font></font></b></td><td>-4</td><td>94</td><td>5</td><td>-25.1</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='green'>A</font><font color='red'>T</font><font color='orange'>G</font><font color='red'>T</font></font></b></td><td>-4</td><td>91</td><td>12</td><td>-17.7</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='red'>T</font><font color='red'>T</font></font></b></td><td>-4</td><td>81</td><td>7</td><td>-18.8</td></tr><tr><td><b><font size='4' face='Courier New'><font color='orange'>G</font><font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font></font></b></td><td>-5</td><td>77</td><td>7</td><td>-18.5</td></tr><tr><td><b><font size='4' face='Courier New'>--<font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='orange'>G</font><font color='red'>T</font><font color='red'>T</font></font></b></td><td>-3</td><td>76</td><td>18</td><td>-10.8</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='blue'>C</font><font color='green'>A</font><font color='blue'>C</font><font color='orange'>G</font><font color='red'>T</font><font color='grey'>N</font><font color='red'>T</font><font color='grey'>N</font><font color='red'>T</font></font></b></td><td>-4</td><td>75</td><td>6</td><td>-18.2</td></tr></table></td><td valign='top'><a href='conocho_2.m0.KSM.txt'><img src='conocho_2.m0.KSM.png' height='300' width='200'></a><br>7.88, hit=158+/0-, auc=63.5</td><td valign='top' align='center'><img src='conocho_2.m0.sequenceHits.png' height='300' width='210'></td><td align='center' valign='bottom'><img src='conocho_2.m0.PWM.png'><br><img src='conocho_2.m0.PWM_rc.png'><br><br><a href='conocho_2.Spatial_dist.m0_m0.txt'><img src='conocho_2.Spatial_dist.m0_m0.png' width='220'></a><p>8.52/13.08, hit=433+/53-, auc=48.5<br></td></tr>
<tr><td align='center'>m1</td><td><table border=1 class="table"><th>K-mer</th><th>Offset</th><th>Pos Hit</th><th>Neg Hit</th><th>HGP</th><tr><td><b><font size='4' face='Courier New'><font color='red'>T</font><font color='red'>T</font><font color='blue'>C</font><font color='red'>T</font><font color='blue'>C</font><font color='blue'>C</font></font></b></td><td>3</td><td>31</td><td>7</td><td>-5.0</td></tr><tr><td><b><font size='4' face='Courier New'>-<font color='red'>T</font><font color='blue'>C</font><font color='red'>T</font><font color='blue'>C</font><font color='blue'>C</font><font color='grey'>N</font><font color='blue'>C</font></font></b></td><td>4</td><td>22</td><td>3</td><td>-5.1</td></tr></table></td><td valign='top'><a href='conocho_2.m1.KSM.txt'><img src='conocho_2.m1.KSM.png' height='300' width='200'></a><br>1.56, hit=49+/7-, auc=11.9</td><td valign='top' align='center'><img src='conocho_2.m1.sequenceHits.png' height='300' width='180'></td><td align='center' valign='bottom'><img src='conocho_2.m1.PWM.png'><br><img src='conocho_2.m1.PWM_rc.png'><br><br><a href='conocho_2.Spatial_dist.m0_m1.txt'><img src='conocho_2.Spatial_dist.m0_m1.png' width='220'></a><p>6.93/9.99, hit=112+/56-, auc=11.1<br></td></tr></table></td></tr></table><p>
Options:
--d /home/bgp01/methylation/programs/gem/Read_Distribution_default.txt --g /home/bgp01/methylation/data/commonData/genome.index.txt --genome /home/bgp01/methylation/data/commonData/Athchrs/ --s 150000000 --expt MYCH7amplified24JAClean8.bam --ctrl /home/bgp01/methylation/data/tfs/Input/amplified/24/JA/Inputamplified24JA.bam --out /home/bgp01/methylation/data/tfs/MYCH7/amplified/24/JA/conocho --f SAM --outNP --range 200 --smooth 0 --mrc 1 --fold 2 --q 1.301029996 --k_min 6 --k_max 20 --k_seqs 600 --k_neg_dinu_shuffle --pp_nmotifs 1 --t 12 <p>